test/e2e_node/remote/remote.go

```diff
@@ -276,7 +276,12 @@ func RunRemote(archive string, host string, cleanup bool, junitFilePrefix string
 		// Insert args at beginning of testArgs, so any values from command line take precedence
 		testArgs = fmt.Sprintf("--experimental-mounter-rootfs-path=%s ", mounterRootfsPath) + testArgs
 		testArgs = fmt.Sprintf("--experimental-mounter-path=%s ", mounterPath) + testArgs
-		glog.Infof("GCI node and GCI mounter both detected, setting --experimental-mounter-path=%q and --experimental-mounter-rootfs-path=%q accordingly", mounterPath, mounterRootfsPath)
+
+		glog.Warningf("Flag forwarding to Kubelet for --experimental-mounter-rootfs-path and --experimental-mounter-path has been disabled." +
```

Comment:
This warning is only needed if node e2e diverges from regular behavior. That's not the case now. So let's remove this.